src/pages/about.mdx

```diff
@@ -36,6 +37,14 @@ At the end of the process, we will produce a report with the outcomes of the ass
 
 [Open Future](https://openfuture.eu/about/) is a European think tank. Its focus is on developing new approaches to an open internet that maximize societal benefits of shared data, knowledge, and culture.
 
+## Partners 
+
+Join our list of committed audience members! Email Shannon at shannonyrshong@gmail.com to join. 
+</Markdown>
+
+<Logos />
+
+<Markdown>
```

Comment:
Most of the "About" page is confined to the narrower container of the `<Markdown>`  component. If you want to see how that works, just accept this change and it will rebuild (we might need to rejig styles for the logos to fit better within the contrained space)

```suggestion

<Logos />

```